test/snyk-to-html.test.ts

```diff
@@ -758,3 +758,81 @@ test('IaC input - all-around test', (t) => {
     },
   );
 });
+
+test('Code input - OpenRefine test', (t) => {
```

Comment:
Can we move Snyk Code-related tests to their own file and assign relevant CODEOWNERS? So that relevant teams are flagged for review if they change.

Reply:
Trying to minimize any changes at the moment - also kept in as with IAC  for now.